app/src/Event/EventFormType.php

```diff
@@ -168,26 +167,26 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'text',
                 [
                     'label' => 'Latitude',
-                    'attr' => ['readonly' => 'readonly'],
+                    'attr'  => ['readonly' => 'readonly'],
                 ]
             )
             ->add(
                 'longitude',
                 'text',
                 [
                     'label' => 'Longitude',
-                    'attr' => ['readonly' => 'readonly'],
+                    'attr'  => ['readonly' => 'readonly'],
                 ]
             )
             ->add(
                 'new_icon',
                 'file',
                 [
                     'data_class' => null,
-                    'label' => 'Upload new icon',
-                    'required' => false,
-                    'attr'=> [
-                        'class'=>'file',
+                    'label'      => 'Upload new icon',
+                    'required'   => false,
+                    'attr'       => [
+                        'class'=> 'file',
```

Comment:
shouldn't there be a space before the => here as well?

Reply:
Probably? But I'm not wanting to hold up the larger change if getting this process automated, then we can tweak the rules to fix things like this. (Or if there aren't rules we can just manually fix)

Reply:
Fair enought we can see to fix this later